server/game/gamesteps/challenge/challengeflow.js

```diff
@@ -221,20 +221,20 @@ class ChallengeFlow extends BaseStep {
         }
 
         this.challenge.claim = this.challenge.getClaim();
-        this.game.promptWithMenu(this.challenge.winner, this, {
+        this.game.promptWithMenu(this.challenge.loser, this, {
             activePrompt: {
-                menuTitle: 'Perform before claim actions',
+                menuTitle: 'Claim will be applied',
                 buttons: [
-                    { text: 'Apply Claim', method: 'applyClaim' },
-                    { text: 'Continue', method: 'cancelClaim' }
+                    { text: 'Continue', method: 'applyClaim' },
+                    { text: 'Cancel', method: 'cancelClaim' }
```

Comment:
I agree with everything here. I basically kept the cancel-option to keep this PR as lean as possible (so as to not detract from my intentions) and because some people mentioned they like it for botched challenges. The latter is probably better fixed by the attackers manually setting their claim to 0, something we can easily communicate to the users via the news. I'll gladly get rid of it if cryogen gives the green light for this PR.